### DISCUSSION TITLE
Artisan 2.10.4 Kaleido M10 Pro AutoDrop

### DISCUSSION THREAD

**Discussion by TheStand70** [artisan-roaster-scope/artisan#1672] [General]
I was roasting 600g of beans to 445°f.
Just after FC @385°f the "drop" happened without touching my laptop. This happened with the SV (B/T max temp) set to 446°f and the beans B/T was 395°. When "drop" is pressed it automatically turn the burner off and the cooling fan on.

I was about 5 ft away from the computer and heard the cooling fan start. Luckly I was able the turn the burner back on fast enough to save the roast. Any idea how the "Drop" function got initiated?

**Comment by roasterdave** [comment]
Most likely Automatic Marking of DROP is enabled (Config>> Events) and a DROP signature was detected or maybe you have an alarm that triggers DROP.  

If you would like me to look deeper to see why DROP triggered in this case, please provide this specific profile .alog file and your .aset settings file (Help>> Save Settings).  Rename each file to end in "**.txt**" then drag-drop the files onto a comment box in this thread here on GitHub.  Do not send in an email reply.

**Reply by TheStand70** [reply]
Didn't save the .alog because the ROR went crazy when it happened. It is connected via bluetooth to the roaster. Could a quick loss of signal indicate a rapid drop in temp and cause this?

And YES automark for drop is on!

**Reply by roasterdave** [reply]
Without a profile to review it is not possible to say what happened or why.

**Comment by Terracotta-6** [comment]
There are two ways to not use the built-in cooling fan. 1. Unplug the fan power plug from the PCB power supply board. 2. Or set it in the software.
![1848de0bcd6494f688fce46425ae358](https://github.com/user-attachments/assets/0a315136-b51e-4a8a-84cf-94e5ec11d175)

**Reply by Terracotta-6** [reply]
Using noise free Armor Type k thermocouple Crazy ROR,  unusable auto DROP .
![4e67acc3faacb7af666581a2ed96362](https://github.com/user-attachments/assets/22e14589-d341-4dac-bcc3-64f0d6ff6d15)
